spyderlib/baseconfig.py

```diff
@@ -204,18 +205,102 @@ def get_image_path(name, default="not_found.png"):
 #==============================================================================
 # Translations
 #==============================================================================
+LANG_FILE = osp.join(get_conf_path(), '.langconfig')
+DEFAULT_LANGUAGE = 'en'
+
+# This needs to be updated every time a new language is added to spyder.
+LANGUAGE_CODES = {'en': u'English',
+                  'fr': u'Français',
+                  'es': u'Español',
+                  'pt_BR': u'Português (brasileiro)'
+                  }
+
+
+def get_available_translations():

```

Comment:
I will take a look a transifex, I had seen it before, but never got the time to go in depth